src/themes/treedip/ProgressBar/ProgressBar.scss

```diff
@@ -0,0 +1,99 @@
+@use '../theme-variables' as v;
+
+$progress-bar-height: v.$progress-bar-height !default;
```

Comment:
Please don't do this, as it's now a fallback the variable can be moved from the theme-variables into the progress bar stylesheet.